xml/admin_ceph_upgrade.xml

```diff
@@ -203,340 +59,448 @@
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Any used software packages have changed significantly.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Special precautions are necessary for your installation.
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    The release notes also provide information that could not make it into the
+    manual on time. They also contain notes about known issues.
+   </para>
+   <para>
+    After having installed the package <package>release-notes-ses</package>,
+    find the release notes locally in the directory
+    <filename>/usr/share/doc/release-notes</filename> or online at
+    <link xlink:href="https://www.suse.com/releasenotes/"/>.
+   </para>
+  </sect2>
+
+  <sect2 xml:id="before-upgrade-password">
+   <title>Verify Your Password</title>
+   <para>
+    Your password must be changed to meet &productname; &productnumber;
+    requirements. Ensure you change the username and password on
+    <emphasis>all</emphasis> initiators as well. For more information on
+    changing your password, see <xref linkend="chap-auth-password"/>.
+   </para>
+  </sect2>
+
+  <sect2 xml:id="before-upgrade-verify-upgrade">
+   <title>Verify the Previous Upgrade</title>
+   <para>
+    In case you previously upgraded from version 4, verify that the upgrade to
+    version 5 was completed successfully:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Check for the existence of the file
+     </para>
+<screen>/srv/salt/ceph/configuration/files/ceph.conf.import</screen>
+     <para>
+      It is created by the engulf process during the upgrade from SES 4 to 5.
+      Also, the <option>configuration_init: default-import</option> option is
+      set in the file
+     </para>
+<screen>/srv/pillar/ceph/proposals/config/stack/default/ceph/cluster.yml</screen>
+     <para>
+      If <option>configuration_init</option> is still set to
+      <option>default-import</option>, the cluster is using
+      <filename>ceph.conf.import</filename> as its configuration file and not
+      &deepsea;'s default <filename>ceph.conf</filename> which is compiled from
+      files in
+     </para>
+<screen>/srv/salt/ceph/configuration/files/ceph.conf.d/</screen>
+     <para>
+      Therefore you need to inspect <filename>ceph.conf.import</filename> for
+      any custom configuration, and possibly move the configuration to one of
+      the files in
+     </para>
+<screen>/srv/salt/ceph/configuration/files/ceph.conf.d/</screen>
+     <para>
+      Then remove the <option>configuration_init: default-import</option> line
+      from
+     </para>
+<screen>/srv/pillar/ceph/proposals/config/stack/default/ceph/cluster.yml</screen>
+     <warning>
+      <title>Default &deepsea; Configuration</title>
       <para>
-       &osd;s
+       If you <emphasis role="bold">do not</emphasis> merge the configuration
+       from <filename>ceph.conf.import</filename> and remove the
```

Comment:
**If** I understand this correctly, would it be clearer to also say "and do not remove the blah option"?